bundle/manifests/sriov-network-operator.clusterserviceversion.yaml

```diff
@@ -333,6 +333,8 @@ spec:
                   value: quay.io/openshift/origin-sriov-network-webhook:4.11
                 - name: SRIOV_INFINIBAND_CNI_IMAGE
                   value: quay.io/openshift/origin-sriov-infiniband-cni:4.11
+                - name: SRIOV_OPERATOR_MUST_GATHER_IMAGE
```

Comment:
SR-IOV must gather image has been deprecated since 4.11. I removed this env var in https://github.com/openshift/sriov-network-operator/pull/670, where did it come out?

Reply:
from here when I run the make bundle it copy the files

https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/59ca039177d1fbf6fea52ab9dd31396c7b988c9a/config/manager/manager.yaml#L73

let me review it only from the bundle and we can have a followup PR on u/s